perfdash/config.go

```diff
@@ -298,6 +298,11 @@ var (
 				OutputFilePrefix: "BenchmarkResults",
 				Parser:           parsePerfData,
 			}},
+			"BenchmarkPerfResults": []TestDescription{{
+				Name:             "",
```

Comment:
why empty name?

Reply:
Perfdash is looking for files prefixed with `$OutputFilePrefix_$Name` string. In our case we generated files prefixed with `BenchmarkPerfScheduling_` followed by a date. I can update https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/util.go#L115 to add `benchmark` string into the file name so we can set the Name to `benchmark` as is done in `BenchmarkResults` case. Though, I don't think it makes any difference.

Reply:
Either do that or simply add a comment here.

Reply:
Done